docs/4.1. Behaviour - Registration.md

```diff
@@ -10,106 +10,107 @@ This document covers expected client and server behaviour for interactions betwe
 
 #### Generic Node Behaviour
 
-The following behaviour assumes a Node with a single network interface for all traffic, or one which uses a single network interface for control and other interfaces for media traffic. It is expected that registrations take place via a single interface in both of these cases.
+The following behaviour assumes a Node with a single network interface for all traffic, or one which uses a single network interface for control and other interfaces for media traffic. Registrations SHOULD happen via a single interface in each of these cases.
 
-1. A Node is connected to the network
+1. A Node is connected to the network.
 2. The Node runs an HTTP accessible Node API.
 3. The Node produces an mDNS advertisement of type '\_nmos-node.\_tcp' in the '.local' domain as specified in [Node API](../APIs/NodeAPI.raml).
 4. The Node performs a DNS-SD browse for services of type '\_nmos-register.\_tcp' as specified in [Discovery: Registered Operation](3.1.%20Discovery%20-%20Registered%20Operation.md).
 5. The Node registers itself with the Registration API by taking the object it holds under the Node API's /self resource and POSTing this to the Registration API.
 6. The Node persists itself in the registry by issuing heartbeats as below.
-7. The Node registers its other resources (from /devices, /sources etc) with the Registration API. Note that resources must be registered in the correct order, such that a Receiver which references a device_id is registered after that corresponding Device (for example).
+7. The Node registers its other resources (from /devices, /sources etc) with the Registration API. Resources MUST be registered in the correct order, such that a Receiver which references a device_id is registered after that corresponding Device (for example).
 
 #### Resilient Node Behaviour
 
-In some deployments, it may be desirable to send multiple copies of essence (video/audio/data) via redundant paths in order to provide guarantees about availability. In these situations, it is likely that a single Node will be used at the point of content creation and transmission via the two paths, and again a single Node used for reception from the two paths.
+In some deployments, multiple copies of essence (video/audio/data) are sent via redundant paths in order to provide guarantees about availability. In these situations, it is likely that a single Node will be used at the point of content creation and transmission via the two paths, and again a single Node used for reception from the two paths.
 
-The data model employed by NMOS allows for this situation by modelling a single Flow sent via two independent Senders which are assigned to separate physical network interfaces on a Node.
+NMOS models this as a single Flow sent via two independent Senders which are assigned to separate physical network interfaces on a Node.
```

Comment:
This has led to confusion for some readers, since although the IS-04 model _allows_ this representation, ST 2022-7 is actually modelled by a _single_ Sender having two `interface_bindings`.

Reply:
True, but out of scope of this PR

Reply:
```suggestion
NMOS can model this as a single Flow sent via two independent Senders which are assigned to separate physical network interfaces on a Node.
```

Reply:
Done